main.js

```diff
@@ -32,90 +148,44 @@ async function run() {
 
     await io.mkdirP(dest);
 
-    const p_release = core.getInput('release') === 'true';
-    const p_update = core.getInput('update') === 'true';
-    const p_pathtype = core.getInput('path-type');
-    const p_msystem = core.getInput('msystem');
-    const p_install = core.getInput('install');
-
-    let base = 'C:';
+    const input = parseInput();
 
-    if (p_release) {
+    let msysRootDir;
```

Comment:
I'd do `let msysRootDir = path.join('C:', 'msys64');` here, and remove the `else`.